subprojects/build-init/src/main/groovy/org/gradle/buildinit/plugins/internal/maven/Maven2Gradle.groovy

```diff
@@ -307,18 +306,12 @@ version = '$project.version'""";
             if (projectDep) {
                 createProjectDependency(projectDep, sb, scope, allProjects)
             } else {
-                def providedMessage = "";
                 if (!war && scope == 'providedCompile') {
-                    scope = 'compile'
-                    providedMessage = '''\
-                       /* This dependency was originally in the Maven provided scope, but the project was not of type war.
-                       This behavior is not yet supported by Gradle, so this dependency has been converted to a compile dependency.
-                       Please review and delete this closure when resolved. */
-                       '''.stripIndent(16)
+                    scope = 'compileOnly'
                 }
                 def exclusions = mavenDependency.exclusions.exclusion
-                if (exclusions.size() > 0 || providedMessage != "") {
-                    createComplexDependency(mavenDependency, sb, scope, providedMessage)
+                if (exclusions.size() > 0) {
+                    createComplexDependency(mavenDependency, sb, scope, "")
```

Comment:
We should simply remove the last parameter from the method signature plus the handling of the `providedMessage` within `createComplexDependency`. It's simply not needed anymore.